modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ExportImportUtil.java

```diff
@@ -0,0 +1,170 @@
+
+/**
+ * @author Guilherme Sá
+ */
+public class ExportImportUtil {
+
+	public static void entityImport(
+			Class<?> clazz, ClassLoader classLoader, File file,
+			MVCActionCommand mvcActionCommand, String entityName,
+			String externalReferenceCode, String jSONName, String sourceFile,
+			Portal portal, User user)
+		throws Exception {
+
+		MockMultipartHttpServletRequest mockMultipartHttpServletRequest =
+			new MockMultipartHttpServletRequest();
+
+		byte[] bytes = file.getBytes(
+			clazz.getResourceAsStream("dependencies/" + sourceFile));
+
+		mockMultipartHttpServletRequest.addFile(
+			new MockMultipartFile(sourceFile, bytes));
+
+		mockMultipartHttpServletRequest.setCharacterEncoding(StringPool.UTF8);
+
+		String boundary = "WebKitFormBoundary" + StringUtil.randomString();
+
+		String start = StringBundler.concat(
+			StringPool.DOUBLE_DASH, boundary,
+			"\r\nContent-Disposition:form-data;name=\"", jSONName,
+			"\";filename=\"", sourceFile,
+			"\";\r\nContent-type:application/json\r\n\r\n");
+		String end = StringBundler.concat(
+			"\r\n--", boundary, StringPool.DOUBLE_DASH);
+
+		mockMultipartHttpServletRequest.setContent(
+			ArrayUtil.append(start.getBytes(), bytes, end.getBytes()));
+
+		mockMultipartHttpServletRequest.setContentType(
+			MediaType.MULTIPART_FORM_DATA_VALUE + "; boundary=" + boundary);
+
+		MockLiferayPortletActionRequest mockLiferayPortletActionRequest =
+			new MockLiferayPortletActionRequest(
+				mockMultipartHttpServletRequest);
+
+		if (Validator.isNotNull(externalReferenceCode)) {
+			mockLiferayPortletActionRequest.addParameter(
+				"externalReferenceCode", externalReferenceCode);
+		}
+
+		mockLiferayPortletActionRequest.addParameter("name", entityName);
+		mockLiferayPortletActionRequest.addParameter(
+			"redirect", RandomTestUtil.randomString());
+		mockLiferayPortletActionRequest.setAttribute(
+			WebKeys.THEME_DISPLAY, _getThemeDisplay(user));
+
+		ReflectionTestUtil.setFieldValue(
+			mvcActionCommand, "_portal",
+			ProxyUtil.newProxyInstance(
+				classLoader, new Class<?>[] {Portal.class},
+				(proxy, method, args) -> {
+					if (Objects.equals(
+							method.getName(), "getUploadPortletRequest")) {
+
+						LiferayPortletRequest liferayPortletRequest =
+							portal.getLiferayPortletRequest(
+								mockLiferayPortletActionRequest);
+
+						return UploadTestUtil.createUploadPortletRequest(
+							portal.getUploadServletRequest(
+								liferayPortletRequest.getHttpServletRequest()),
+							liferayPortletRequest,
+							portal.getPortletNamespace(
+								liferayPortletRequest.getPortletName()));
+					}
+
+					return method.invoke(portal, args);
+				}));
+
+		mvcActionCommand.processAction(
+			mockLiferayPortletActionRequest,
+			new MockLiferayPortletActionResponse());
+	}
+
+	public static void validateEntityExport(
+			Class<?> clazz, String compareFile, long entityId,
+			String entityIdentificator, MVCResourceCommand mvcResourceCommand,
```

Comment:
Rename to:

- `compareFile` -> `fileName`
- `entityId` -> `id`
- `entityIdentificator` -> `identifier`

Reply:
See [this comment](https://github.com/liferay-objects/liferay-portal/pull/2939#discussion_r1427123862)